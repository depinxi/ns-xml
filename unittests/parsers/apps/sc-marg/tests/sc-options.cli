sub -d abc def ghi "This should be ok"