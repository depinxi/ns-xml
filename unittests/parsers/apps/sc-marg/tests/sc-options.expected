CLI: "sub-command", "-d", "abc", "def", "ghi", "This should be ok"
Value count: 3
Values: "def", "ghi", "This should be ok"
Error count: 0
Subcommand: sub-command
switchA=False
argumentB=
multiargC=
sub-command_argumentD=abc
sub-command_switchE=False
sub-command_switchA2=False
